tests/test_cmdline.py

```diff
@@ -446,13 +454,15 @@ def test_list_sources__in_varied_project__lists_all_files(fake_project):
         str(tmp_path / filename)
         for filename in [
             "code.py",
-            "subdir/other.py",
-            "subdir/notebook.ipynb",
+            os.path.join("subdir", "other.py"),
+            os.path.join("subdir", "notebook.ipynb"),
             "requirements.txt",
             "pyproject.toml",
             "setup.py",
             "setup.cfg",
-            f"my_venv/lib/python{major}.{minor}/site-packages",
+            os.path.join("my_venv", "Lib", "site-packages")
+            if platform.system() == "Windows"
+            else f"my_venv/lib/python{major}.{minor}/site-packages",
```

Comment:
I think I find it easier to read when the Windows-condition is not buried inside an if-else expression inside a list comprehension:

```python
    output, returncode = run_fawltydeps_function("--list-sources", f"{tmp_path}")

    if sys.platform.startswith("win"):  # Windows
        site_packages = Path("my_venv", "Lib", "site-packages")
    else:  # Assume POSIX
        major, minor = sys.version_info[:2]
        site_packages = Path("my_venv", "lib", f"python{major}.{minor}", "site-packages")
    expect = [
        str(tmp_path / filename)
        for filename in [
            "code.py",
            os.path.join("subdir", "other.py"),
            os.path.join("subdir", "notebook.ipynb"),
            "requirements.txt",
            "pyproject.toml",
            "setup.py",
            "setup.cfg",
            str(site_packages),
        ]
    ]
    assert_unordered_equivalence(output.splitlines()[:-2], expect)
    assert returncode == 0
```

Reply:
Already fixed by using `site_packages()` helper.